src/backend/base/langflow/api/v1/mcp_projects.py

```
@@ -399,14 +401,64 @@
+                                    config_path = (
+                                        user_dirs[0] / "AppData" / "Roaming" / "Claude" / "claude_desktop_config.json"
+                                    )
+                                else:
+                                    raise HTTPException(
+                                        status_code=400, detail="Could not find Windows user directory in WSL"
+                                    )
+                            else:
+                                raise HTTPException(
+                                    status_code=400, detail="Windows C: drive not mounted at /mnt/c in WSL"
+                                )
+                    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)